### PR TITLE
Changes Done: HTML Assignments(1,2,3,4)

### PR DESCRIPTION
**Pull request by anuja1603**

### CODE REVIEW

**Review by Amit727619 — APPROVED** [review]

**Review by pooja-nagare — APPROVED** [review]

**Review by pritikekan — APPROVED** [review]

**Review by shivani-pawbake — APPROVED** [review]

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]
Why you added script file in assignments1 please remove it.

**Comment by anuja1603** [comment]
> Why you added script file in assignments1 please remove it.

> Please check it. I removed that line.

**Review by Priyankashelke2411 — APPROVED** [review]

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]
remove extra lines

File: Anuja/HTML/ASSN1/Que1.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Question number 1</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <script src='main.js'></script>
```
Comment (Priyankashelke2411) [inline Anuja/HTML/ASSN1/Que1.html]: remove script file

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

File: Anuja/HTML/ASSN3/MI-MEAN.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<style>
+    body{
+        background-color: pink;
+    }
+</style>
+
```
Comment (Priyankashelke2411) [inline Anuja/HTML/ASSN3/MI-MEAN.html]: remove extra line here

File: Anuja/HTML/ASSN3/Contact-Us.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <a href="MI-MEAN.html" >MI-MEAN </a> |
+        <a href="Contact-Us.html" >Contact-Us </a> |
+    </div>
+    <div style="display: flex;">
+        <div>
+            <img src="1.jpeg" alt="Contact-Us" height="400px;" width="400px;">
+        </div>
+        <p>Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.vContact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.</p>
+    </div>
+
```
Comment (Priyankashelke2411) [inline Anuja/HTML/ASSN3/Contact-Us.html]: remove extra line

File: Anuja/HTML/ASSN1/Que1.html (hunk 1 above)
Comment (anuja1603) [reply Anuja/HTML/ASSN1/Que1.html]: Mam I have removed that.

**Review by Priyankashelke2411 — APPROVED** [review]

File: Anuja/HTML/ASSN1/Que1.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    <title>Question number 1</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+</head>
+<body>
+    <h1>HTML</h1>
+    <h3>Introduction</h3>
+    <p>HTML is Standard markup language for Web Pages. </p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que1.html]: check indentation

Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que1.html]: add empty newline at end of file

File: Anuja/HTML/ASSN1/Que2.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    <h1>HTML</h1>
+    <h2>Introduction</h2>
+    <p>HTML is a standard markup language for Web Pages.</p>
+    <h3>HTML Editor</h3>
+    <h4>Notepad or TextEditor</h4>
+    <p>Web Pages can be created and modified by using profesion HTML editors.<br>
+        However, for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(Mac).
+    </p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que2.html]: add empty newline at end of file

Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que2.html]: check indentation

File: Anuja/HTML/ASSN1/Que3.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    <h2>Introduction</h2>
+    <p>HTML is a standard markup language for Web Pages.</p>
+    <hr>
+    <h3>HTML Editor</h3>
+    <h4>Notepad or TextEdit.</h4>
+    <p>Web Pages can be created and modified by using professional HTML editors.<br>
+        However, for learning HTML we recommend a simple text editor like Notepad(PC) or TextEdit(Mac).
+    </p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que3.html]: add empty newline at end of file

Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que3.html]: check indent

File: Anuja/HTML/ASSN1/Que4.html [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Introduction Page</title>
+</head>
+<style>
+    body{
+        text-align: center;
+        color: white;
+        background-color: black;
+
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que4.html]: remove unwanted lines

File: Anuja/HTML/ASSN1/Que4.html [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+<body>
+    <h1 style="font-size: 70px; font-family: cursive;">Anuja Vinod Pardeshi</h1>
+    <p style="font-size: 20px; padding: 15px;">I have recently completed my graduation in "Computer Enginnering" from SSVPS BSD College of Engineering Dhule, Maharashtra.<br>
+        I have a keen intrest in Full stack Development and AI Techonologies.<br> I have done Web Development and designing Course and Python Programming Course.<br>
+        My hobbies are Tavelling and Painting. <br>
+        I'm passionate about Photography.
+    </p>
+
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN1/Que4.html]: check indentation

File: Anuja/HTML/ASSN2/Que1.html [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    </p>
+    <hr>
+    <h3>Initial public offering</h3>
+    <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.
+        At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 <u>20</u>
+        years, until the year 2024.
+    </p>
+    <p style="text-align: center; color: green; font-weight: bold; font-size: 20px;">Thank you!!!</p>
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN2/Que1.html]: add empty newline at end of file

Comment (JafarPathan) [inline Anuja/HTML/ASSN2/Que1.html]: check indentation

File: Anuja/HTML/ASSN3/Contact-Us.html [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    </div>
+    <div style="display: flex;">
+        <div>
+            <img src="1.jpeg" alt="Contact-Us" height="400px;" width="400px;">
+        </div>
+        <p>Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.vContact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.Contact-Us.</p>
+    </div>
+    
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN3/Contact-Us.html]: remove unwanted lines and check indentation

File: Anuja/HTML/ASSN3/MI-MEAN.html [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+        <li>Haruday</li>
+        <li>Aniket</li>
+    </ul>
+    <ol style="list-style-type: lower-roman;">
+        <li>Bipin</li>
+        <li>Priyanka</li>
+        <li>Yashvini</li>
+    </ol>
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN3/MI-MEAN.html]: add empty newline at end of file

Comment (JafarPathan) [inline Anuja/HTML/ASSN3/MI-MEAN.html]: remove unwanted lines and check indentation

File: Anuja/HTML/ASSN3/Que1.html [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+            <li>SYSTEM-2</li>
+            <ul>
+                <li>Triveni</li>
+                <li>Kalyani</li>
+            </ul>
+        </ol>
+    </ol>
+    
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN3/Que1.html]: add empty newline at end of file

Comment (JafarPathan) [inline Anuja/HTML/ASSN3/Que1.html]: remove unwanted lines and check indentation

File: Anuja/HTML/ASSN3/Que2.html [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+        <div>
+            <img src="2.png" alt="image" height="400px" width="400px">
+        </div>
+    </div>
+    <div>
+        <h2 style="text-align: left;">Mean Stack Development</h2>
+        <p>This program will advance your career as a MEAN stack developer. You’ll learn top skills such as MongoDB, Express.js, Angular, and Node.js (“MEAN”), plus GIT, HTML, CSS, and JavaScript to build and deploy interactive applications and services.</p>
+    </div>
+</body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN3/Que2.html]: check indentation

File: Anuja/HTML/ASSN4/Que1.html [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+      </tr>
+      <tr>
+        <th>Females</th>
+        <td>1.7</td>
+        <td>0.007</td>
+        <td>43%</td>
+      </tr>
+    </table>
+  </body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN4/Que1.html]: remove unwanted lines and check indentation

Comment (JafarPathan) [inline Anuja/HTML/ASSN4/Que1.html]: add empty newline at end of file

File: Anuja/HTML/ASSN4/Que2.html [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+      </tr>
+      <tr>
+        <th>4:30</th>
+        <td></td>
+        <td></td>
+        <td></td>
+      </tr>
+    </table>
+  </body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN4/Que2.html]: add empty newline at end of file

Comment (JafarPathan) [inline Anuja/HTML/ASSN4/Que2.html]: remove unwanted lines and check indentation

File: Anuja/HTML/ASSN4/Que3.html [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+
+      <tr>
+        <td>Wednesday</td>
+        <td class="yellow">8:00 a.m.</td>
+        <td class="green">12:00 p.m.</td>
+        <td class="center">XSL Formatting</td>
+      </tr>
+    </table>
+  </body>
+</html>
```
Comment (JafarPathan) [inline Anuja/HTML/ASSN4/Que3.html]: remove unwanted lines and check indentation

Comment (JafarPathan) [inline Anuja/HTML/ASSN4/Que3.html]: add empty newline at end of file

**Review by JafarPathan — APPROVED** [review]

**Pull request merged**